WPMedia/fusion-news-theme-blocks
# Tmedia 516 revert arc ads 6

## Description
Revert arcads 6 as testing does not work

## Jira Ticket
- [TMEDIA-516](https://arcpublishing.atlassian.net/browse/TMEDIA-516)

## Acceptance Criteria
Tests should pass 

## Test Steps

1. Checkout this branch `git checkout TMEDIA-516-revert-arc-ads-6`
2. `rm -rf node_modules && npm i && npm`
3. Tests should pass

## Effect Of Changes
### Before
Tests didn't pass after the linked package was not updated in the root package-lock

```
    Failed to get mock metadata: /Users/howardj/sites/fusion-news-theme-blocks/node_modules/arcads/dist/arcads.js

    See: https://jestjs.io/docs/manual-mocks.html#content

```

### After
Tests pass as the upgraded version does not have tests so we can't verify it works as expected in our test cases https://arcpublishing.atlassian.net/browse/ACI-308

## Dependencies or Side Effects
- none

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored). -> tests didn't work so that's why this should be pulled
- [x] Confirmed relevant documentation has been updated/added. -> updated the docs referring to version 6

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
